### DISCUSSION TITLE
redirect requests from multiple domains

### DISCUSSION THREAD

**Discussion by coaperator** [nginx-proxy/nginx-proxy#1926] [Q&A]
Hey!
I have 4 domains
```
site.com
site-2.com
super-site.com
site-super.com
```
I configured my application to work with the first domain - `site.com`
how can i make nginx 301 redirect from other three domains `site-2.com` `super-site.com` `site-super.com` to `site.com`
over both http and https protocols?

My docker file with settings is now like this
```
FROM jwilder/nginx-proxy

RUN { \
      echo 'client_max_body_size 100m;'; \
    } > /etc/nginx/conf.d/custom_proxy.conf
```

**Comment by coaperator** [comment]
UP

**Comment by tkw1536** [comment]
There is a PR adding this feature: #1369.
It currently needs some tests in order to be merged.

**Reply by tkw1536** [reply]
A workaround might be to have two application containers - one for just `site.com`, which serves your application.
And a second one for `site-2.com,super-site.com,site-super.com` that serves a server that only redirects. 

For the latter I personally use https://github.com/tkw1536/tr, but there might be other possibilities.

**Reply by coaperator** [reply]
no it's not an option
do you know when the next pull request merge is expected?
there are a lot of good ones that are very necessary in everyday use

**Comment by coaperator** [comment]
UP, looking forward to this feature
